coreAPI/src/main/java/net/java/games/input/DefaultControllerEnvironment.java

```diff
@@ -153,14 +157,23 @@ public Controller[] getControllers() {
         }
         return ret;
     }
-    
-    /* This is jeff's new plugin code using Jeff's Plugin manager */
+
+	@Override
+	public void release() {
+        this.loadedPlugins.forEach(it -> it.release());
+
+        this.controllers = null;
+        this.controllerListeners.clear();
+		return;
+	}
+
+	/* This is jeff's new plugin code using Jeff's Plugin manager */
```

Comment:
This comment can probably be removed, unless it's important to know that it's using Jeff's Plugin manager?

Reply:
That wasn't added by me, it was there before and I'm not quite sure why. So I'll just leave it